qual_id/categories/electronic.py

```diff
@@ -0,0 +1,17 @@
+from ..category import Category
+
+
+class Electronic(Category):
+  def get_values(self):
+    return [
+        'laptop',
+        'Desktop',
+        'refrigerator',
+        'mobile',
+        'television',
+        'radio',
+        'trimmer',
+		'headPhones',
```

Comment:
```suggestion
	'headphones',
```
No need for camelcase